### PR TITLE
Nerf +Insist

### PR DESCRIPTION
**Pull request by DeathlyPlays**
This is meant to be added BEFORE abilities project is done

### CODE REVIEW

**Comment by HoeenCoder** [comment]
That will cause conflicts, also you could of just changed the branch back instead of closing the PR.

File: mods/sgssb/moves.js [hunk 1]
```diff
@@ -616,6 +616,7 @@ exports.BattleMovedex = {
 		name: "Aqua Subscribe",
 		priority: 1,
 		self: {
+			chance: 30,
```
Comment (HoeenCoder) [inline mods/sgssb/moves.js:619]: I'm not sure this will work.

Comment (Eelektross) [inline mods/sgssb/moves.js]: secondary: {
			chance: 30,
			self: {
				boosts: {
					spa: 1,
spe: 1,
				},
			},

Comment (Eelektross) [inline mods/sgssb/moves.js]: thats a better way of writing your boost chance

**Review by ghost — CHANGES_REQUESTED** [review]
There is a better way to code this:

`secondary: {
			chance: 30,
			self: {
				boosts: {
					spa: 1,
spe: 1,
				},
			},`

File: mods/sgssb/moves.js (hunk 1 above)
Comment (DeathlyPlays) [reply mods/sgssb/moves.js:619]: It does

Comment (HoeenCoder) [reply mods/sgssb/moves.js:619]: Ill test it later

Comment (DeathlyPlays) [reply mods/sgssb/moves.js:619]: Ok

File: mods/sgssb/moves.js [hunk 2]
```diff
@@ -615,7 +615,8 @@ exports.BattleMovedex = {
 		id: "aquasubscribe",
 		name: "Aqua Subscribe",
 		priority: 1,
-		self: {
+		secondary: {
```
Comment (HoeenCoder) [inline mods/sgssb/moves.js]: I think youll need to specify `target: "self"` in the secondary so it dosent boost the foe.

Comment (DeathlyPlays) [reply mods/sgssb/moves.js]: Oh yeah.... Just using self works though haha but I made it secondary for your sake

Comment (DeathlyPlays) [reply mods/sgssb/moves.js]: Im on mobile so I cant add that as well bc it wont tab right

**Comment by DeathlyPlays** [comment]
Meh this works so Ill keep it simple

File: mods/sgssb/moves.js [hunk 3]
```diff
@@ -615,7 +615,7 @@ exports.BattleMovedex = {
 		id: "aquasubscribe",
 		name: "Aqua Subscribe",
 		priority: 1,
-		secondary: {
+		self: {
```
Comment (HoeenCoder) [inline mods/sgssb/moves.js]: Thats not what i meant....

I mean add ``target: "self",`` INSIDE the secondary object

Comment (DeathlyPlays) [reply mods/sgssb/moves.js]: Ik but this works

Comment (DeathlyPlays) [reply mods/sgssb/moves.js]: It's been tested

Comment (HoeenCoder) [reply mods/sgssb/moves.js]: Regardless, I need to test this before I implement, a formats crash requires a restart most of the time.

**Comment by HoeenCoder** [comment]
Closed because no longer needed.

**Pull request closed without merging**